### ISSUE TITLE
Assumes "Assets" container exists

### ISSUE THREAD

**Issue by jolora** [alt-design/Alt-SEO-Addon#29]
I've just installed this addon into a relataively fresh 5.2 install. When I attempt to access /cp/alt-design/alt-seo I get the following error:

```
Statamic \ Exceptions \ AssetContainerNotFoundException PHP 8.3.2 11.7.0
Asset Container [assets] not found
```

It looks like the addon is assuming I have an "assets" container. I only have an "images" container.

**Comment by Alt-Ben** [comment]
Hey @jolora! So you can change the asset container it's looking for on the settings page (look for the Alt SEO icon on the left hand side), there's a field for the asset container name, just pop yours in there and it should sort it!

Let me know if not!

**Comment by jolora** [comment]
Thanks. Problem was the error stopped me accessing the settings at all. I can just create a container with that name but thought you'd probably still want to know about the issue. Thanks for sharing a free addon btw.

**Comment by Alt-Ben** [comment]
Ah interesting - we'll take a look! Might be a Statamic v5 issue we haven't come across. Thanks for letting me know 🙌

**Comment by Alt-Ben** [comment]
Fixed in v1.1.8!
